clientgui/ViewStatistics.cpp

```diff
@@ -739,8 +739,11 @@ void CPaintStatistics::DrawAxis(wxDC &dc, const double max_val_y, const double m
 	double y_start_val = ceil(min_val_y / d_oy_val) * d_oy_val;
 	d_oy_count = (int)floor((max_val_y - y_start_val) / d_oy_val);
 
+    double temp_int_part;
+    int prec = modf(d_oy_val, &temp_int_part) == 0.0 ? 0 : 2;
```

Comment:
How does this cope with values like `5.001`? Shouldn't that set precision to 0? Instead it is set to 2.

Reply:
Ok. Will do

Reply:
Fixed